--- javax/servlet/ServletOutputStream.java	2008-06-06 11:04:48.000000000 +1000
+++ ../original-sources/javax/servlet/ServletOutputStream.java	2008-09-24 10:28:37.000000000 +1000
@@ -31,6 +31,7 @@
 import java.io.IOException;
 import java.io.CharConversionException;
 import java.text.MessageFormat;
+import java.util.PropertyResourceBundle;
 import java.util.ResourceBundle;
 
 /**
@@ -52,10 +53,20 @@
 
 public abstract class ServletOutputStream extends OutputStream {
 
-    private static final String LSTRING_FILE = "javax.servlet.LocalStrings";
-    private static ResourceBundle lStrings =
-	ResourceBundle.getBundle(LSTRING_FILE);
+    private static final String LSTRING_FILE = "/javax/servlet/LocalStrings.properties";
+    private static ResourceBundle lStrings;
 
+   static
+   {
+      try
+      {
+           lStrings = new PropertyResourceBundle(ServletOutputStream.class.getResourceAsStream(LSTRING_FILE));
+      }
+      catch (Exception e)
+      {
+           e.printStackTrace();
+      }
+   }
 
     
     /**
