--- javax/servlet/http/Cookie.java	2008-06-06 11:04:46.000000000 +1000
+++ ../original-sources/javax/servlet/http/Cookie.java	2008-09-24 10:20:44.000000000 +1000
@@ -28,6 +28,7 @@
 package javax.servlet.http;
 
 import java.text.MessageFormat;
+import java.util.PropertyResourceBundle;
 import java.util.ResourceBundle;
 
 /**
@@ -75,9 +76,21 @@
 public class Cookie implements Cloneable {
 
     private static final String LSTRING_FILE =
-	"javax.servlet.http.LocalStrings";
-    private static ResourceBundle lStrings =
-	ResourceBundle.getBundle(LSTRING_FILE);
+	"/javax/servlet/http/LocalStrings.properties";
+    private static ResourceBundle lStrings;
+
+    static 
+    {
+        try
+        {
+            lStrings = new PropertyResourceBundle(Cookie.class.getResourceAsStream(LSTRING_FILE));
+        }
+        catch (Exception e)
+        {
+            e.printStackTrace();
+        }
+
+    }
     
     //
     // The value of the cookie itself.
