--- javax/servlet/http/HttpServlet.java	2008-06-06 11:04:46.000000000 +1000
+++ ../original-sources/javax/servlet/http/HttpServlet.java	2008-09-24 10:21:57.000000000 +1000
@@ -36,6 +36,7 @@
 import java.text.MessageFormat;
 import java.util.Enumeration;
 import java.util.Locale;
+import java.util.PropertyResourceBundle;
 import java.util.ResourceBundle;
 
 import javax.servlet.GenericServlet;
@@ -104,10 +105,21 @@
     private static final String HEADER_LASTMOD = "Last-Modified";
     
     private static final String LSTRING_FILE =
-	"javax.servlet.http.LocalStrings";
-    private static ResourceBundle lStrings =
-	ResourceBundle.getBundle(LSTRING_FILE);
+	"/javax/servlet/http/LocalStrings.properties";
+    private static ResourceBundle lStrings;
    
+    static
+    {
+        try
+        {
+            lStrings = new PropertyResourceBundle(HttpServlet.class.getResourceAsStream(LSTRING_FILE));
+        }
+        catch (Exception e)
+        {
+            e.printStackTrace();
+        }
+    }
+
    
    
     
