--- /home/janb/src/jetty-6.1/modules/jetty/src/main/java/org/mortbay/jetty/MimeTypes.java	2007-12-21 13:53:51.000000000 +1100
+++ org/mortbay/jetty/MimeTypes.java	2008-02-20 13:47:07.000000000 +1100
@@ -14,13 +14,17 @@
 
 package org.mortbay.jetty;
 
+import java.io.IOException;
 import java.util.Enumeration;
 import java.util.HashMap;
 import java.util.Iterator;
 import java.util.Map;
 import java.util.MissingResourceException;
+import java.util.PropertyResourceBundle;
 import java.util.ResourceBundle;
 
+import org.mortbay.ijetty.IJettyService;
+import org.mortbay.ijetty.R;
 import org.mortbay.io.Buffer;
 import org.mortbay.io.BufferCache;
 import org.mortbay.io.BufferCache.CachedBuffer;
@@ -92,7 +96,7 @@
     {
         try
         {
-            ResourceBundle mime = ResourceBundle.getBundle("org/mortbay/jetty/mime");
+            ResourceBundle mime = new PropertyResourceBundle(IJettyService.getStreamToRawResource(R.raw.org_mortbay_jetty_mime));
             Enumeration i = mime.getKeys();
             while(i.hasMoreElements())
             {
@@ -106,10 +110,15 @@
             Log.warn(e.toString());
             Log.debug(e);
         }
+        catch (IOException e)
+        {
+            Log.warn(e.toString());
+            Log.debug(e); 
+        }
 
         try
         {
-            ResourceBundle encoding = ResourceBundle.getBundle("org/mortbay/jetty/encoding");
+            ResourceBundle encoding = new PropertyResourceBundle(IJettyService.getStreamToRawResource(R.raw.org_mortbay_jetty_encoding));
             Enumeration i = encoding.getKeys();
             while(i.hasMoreElements())
             {
@@ -122,7 +131,11 @@
             Log.warn(e.toString());
             Log.debug(e);
         }
-
+        catch (IOException e)
+        {
+            Log.warn(e.toString());
+            Log.debug(e); 
+        }
         
         TEXT_HTML_BUFFER.setAssociate("ISO-8859-1",TEXT_HTML_8859_1_BUFFER);
         TEXT_HTML_BUFFER.setAssociate("ISO_8859_1",TEXT_HTML_8859_1_BUFFER);
