--- org/mortbay/jetty/MimeTypes.java	2008-09-24 11:42:09.000000000 +1000
+++ ../original-sources/org/mortbay/jetty/MimeTypes.java	2008-09-24 11:44:00.000000000 +1000
@@ -14,11 +14,13 @@
 
 package org.mortbay.jetty;
 
+import java.io.IOException;
 import java.util.Enumeration;
 import java.util.HashMap;
 import java.util.Iterator;
 import java.util.Map;
 import java.util.MissingResourceException;
+import java.util.PropertyResourceBundle;
 import java.util.ResourceBundle;
 
 import org.mortbay.io.Buffer;
@@ -92,7 +94,8 @@
     {
         try
         {
-            ResourceBundle mime = ResourceBundle.getBundle("org/mortbay/jetty/mime");
+            java.io.InputStream is = MimeTypes.class.getResourceAsStream("/org/mortbay/jetty/mime.properties");
+            ResourceBundle mime = new PropertyResourceBundle(is);
             Enumeration i = mime.getKeys();
             while(i.hasMoreElements())
             {
@@ -101,7 +104,7 @@
                 __dftMimeMap.put(StringUtil.asciiToLowerCase(ext),normalizeMimeType(m));
             }
         }
-        catch(MissingResourceException e)
+        catch(Exception e)
         {
             Log.warn(e.toString());
             Log.debug(e);
@@ -109,7 +112,8 @@
 
         try
         {
-            ResourceBundle encoding = ResourceBundle.getBundle("org/mortbay/jetty/encoding");
+            java.io.InputStream is = MimeTypes.class.getResourceAsStream("/org/mortbay/jetty/encoding.properties");
+            ResourceBundle encoding = new PropertyResourceBundle(is);
             Enumeration i = encoding.getKeys();
             while(i.hasMoreElements())
             {
@@ -117,7 +121,7 @@
                 __encodings.put(type,encoding.getString(type.toString()));
             }
         }
-        catch(MissingResourceException e)
+        catch(Exception e)
         {
             Log.warn(e.toString());
             Log.debug(e);
